src/main/java/hw6/DataProviderJsonForMetalsColorPage.java

```diff
@@ -0,0 +1,79 @@
+package hw6;
+
+import hw6.entities.Summary;
+
+import java.util.List;
+import java.util.Objects;
+
+public class DataProviderJsonForMetalsColorPage {
```

Comment:
This is not the DP

Reply:
This is really not a date provider, the date provider itself is in a different place, which I wrote about below. 

I will rename this class so as not to mislead anyone.

Reply:
I think the title "DataFromJsonForMetalsColorPage" will be very useful here